nextflow.config

```diff
@@ -220,7 +221,7 @@ manifest {
     description     = 'Nextflow RNA-Seq analysis pipeline, part of the nf-core community.'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.04.0'
-    version         = '3.2'
+    version         = '3.3dev'
```

Comment:
should be `3.3`? Or is there some CI magic happening at some point?

Reply:
Already resolved in https://github.com/nf-core/rnaseq/pull/673